### PR TITLE
Add XUnit, Moq and simple test

### PR DESCRIPTION
**Pull request by Elinos**

### CODE REVIEW

File: WebApp.API.Tests/ControllerTests/AccountControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using Microsoft.Extensions.Logging;
+using WebApp.API.Models.AccountViewModels;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebApp.API.Tests.ControllerTests
+{
+    public class AccountControllerTests
+    {
+        private readonly UserManager<ApplicationUser> userManagerMock = new FakeUserManager();
```
Comment (ivopashov) [inline WebApp.API.Tests/ControllerTests/AccountControllerTests.cs]: Can we put those in an initializer?

File: WebApp.API.Tests/ControllerTests/AccountControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            //Assert
+            AccountController controller = new AccountController(userManagerMock, accountServiceMock.Object, emailSenderMock.Object, loggerFactoryMock.Object);
+
+            //Act
+            var result = await controller.Register(new RegisterViewModel());
+            var okResult = result as OkResult;
+
+            //Assert
+            Assert.NotNull(okResult);
+            Assert.Equal(200, okResult.StatusCode);
```
Comment (ivopashov) [inline WebApp.API.Tests/ControllerTests/AccountControllerTests.cs]: I would better assert that `_userManager.CreateAsync` gets called with the correct params. IMHO the asserting the status code doesn't bring much value here.

File: WebApp.API.Tests/ControllerTests/AccountControllerTests.cs (hunk 1 above)
Comment (Elinos) [inline WebApp.API.Tests/ControllerTests/AccountControllerTests.cs]: I can put it in a constructor or fixture, but I won't add more tests here, so there is no point

File: WebApp.API.Tests/ControllerTests/AccountControllerTests.cs (hunk 2 above)
Comment (Elinos) [inline WebApp.API.Tests/ControllerTests/AccountControllerTests.cs]: It was a simple test, just to see if it's working

**Pull request merged**